app/screens/post_options/post_options.js

```diff
@@ -261,11 +283,12 @@ export default class PostOptions extends PureComponent {
     getOthersPostOptions = () => {
         const actions = [
             this.getReplyOption(),
-            this.getFlagOption(),
             this.getAddReactionOption(),
-            this.getPinOption(),
+            this.getMarkAsUnreadOption(),
```

Comment:
I thought we were going to keep both of these options the same?

Reply:
Sorry, I'm not sure I follow what you mean by keeping both the same. If you are referring to ordering, I thought we agreed on what is shown [here](https://projects.invisionapp.com/share/N4RBXGMY9QR#/screens/356105929)

Reply:
So we can combine others people post and own posts

Reply:
ah, didn't get that from our conversation, sure I can combine them! 👍